mkosi/qemu.py

```diff
@@ -384,8 +381,11 @@ def start_virtiofsd(
         # as root without the source directories necessarily being owned by root. We achieve this by running
         # virtiofsd as the owner of the source directory and then mapping that uid to root.
 
-        name = f"mkosi-virtiofsd-{name}"
-        description = f"virtiofsd for {directory}"
+        if name is None:
+            name = systemd_escape(config, directory, path=True)
+
+        name = f"mkosi-virtiofsd-{config.machine_or_name()}-{name}"
```

Comment:
Looking at this again, this seems a bit weird? We have five call sites for `start_virtiofsd`, four of them don't set `name`, which means it gets interpolated as `None`, and the one that does, sets it to `config.machine_or_name()`, which would mean that it gets here twice, once escaped and once unescaped.